easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.6-foss-2023a.eb

```diff
@@ -0,0 +1,153 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>; Guillermo AR Piccoli <guillermo.piccoli@hpcnow.com>
+# License::     GPL-v3.0
+# #
```

Comment:
huh?

Reply:
Is that header alright? should I include it? These are all updated unsubmitted recipes that were developed internally by HPCNow! before, that's why Jordi is there,
Cheers